test/test_tcti.py

```diff
@@ -223,6 +223,10 @@ def test_custom_pytcti_finalize_error(self):
         with self.assertRaises(RuntimeError, msg="Bills Error 2"):
             t.finalize()
 
+    def test_is_available(self):
+        self.assertTrue(TCTILdr.is_available("swtpm"))
```

Comment:
You could try using `self.tpm.libname` here, as some tests against older versions of tpm2-tss doesn't use swpm

Reply:
I'm trying with `None` as it should gets the default, at least I hope.

Reply:
looks like it worked.